index.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <script src="https://kit.fontawesome.com/5e67ba493b.js" crossorigin="anonymous"></script>
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <title>Document</title>
+</head>
+<body>
+    <header>
+        <div class="logo"><h1>Welcome</h1></div>
+        <div class="menu-burger">
+            <div></div>
+            <div></div>
+            <div></div>
+        </div>
+    </header>
+    <Main>
+         <div class="info-bio">
+            <h1>Hey there. I’m Heldricks <br>
+                <span class="inner-text">I’m a software developer</span> 
+            </h1>
+            <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                don’t hestiate to contact me.</p>
+         </div>
+         <div class="social">
+            <a href="https://twitter.com/nerdy_me_"><i class="fa-brands fa-twitter"></i></a>
+            <a href="https://github.com/Adholah96"><i class="fa-brands fa-github"></i></a>
+            <a href="https://linkedin.com/in/heldricks-arthur-59ab2411a"><i class="fa-brands fa-linkedin-in"></i></a>
+            <a href=""><i class="fa-solid fa-m"></i></a>
+            <a href=""><img src="./assets/angellist.png" alt=""></a>
+         </div>
```

Comment:
- [ ] I would advise you to use ul and wrap the a tags in li tags individually for semantic and orderliness of the code.